BigBaseV2/src/backend/looped/player/spectate.cpp

```diff
@@ -18,8 +18,9 @@ namespace big
 
 				NETWORK::NETWORK_SET_IN_SPECTATOR_MODE(false, -1);
 				HUD::SET_MINIMAP_IN_SPECTATOR_MODE(false, -1);
-
+				ENTITY::SET_ENTITY_INVINCIBLE(PLAYER::PLAYER_PED_ID(), false);
```

Comment:
Remove set invincible to prevent confusion to users (they don't know their entity is made invincible / vincible).